src/main/java/wedlog/address/model/person/UniqueGuestList.java

```diff
@@ -97,6 +97,31 @@ public void setGuests(List<Guest> guests) {
         internalList.setAll(guests);
     }
 
+    /**
+     * Returns the number of guests with each rsvp status.
+     * The return value is an array of integers with each entry representing each of the 3 rsvp statuses.
+     * @return integer array of rsvp status counts.
+     */
+    public int[] getRsvpStatusCounts() {
+        int[] count = new int[3];
+        for (Guest guest : internalList) {
+            switch (guest.getRsvpStatus().value) {
+            case "yes":
+                count[0]++;
+                break;
+            case "no":
+                count[1]++;
+                break;
+            case "unknown":
+                count[2]++;
+                break;
+            default:
+                break;
+            }
+        }
+        return count;
+    }
+
```

Comment:
Perhaps we could create some sort of abstraction or class for rsvp statistics?

Reply:
yep